apis/python/setup.py

```diff
@@ -82,6 +80,9 @@ def libtiledbsoma_exists():
 
 
 def find_or_build_package_data(setuptools_cmd):
+    this_dir = pathlib.Path(__file__).parent.absolute()
```

Comment:
maybe move this under `# Set up paths` so that it’s right next to `scripts_dir`?